### PR TITLE
Fix grammar in first sentence of local-dom.md

### PR DESCRIPTION
**Pull request by BenoitPerrot**
Say "an element that is" instead of "that an element is"

### CODE REVIEW

File: 1.0/docs/devguide/local-dom.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-We call the DOM that an element is in charge of creating and managing its _local
+We call the DOM an element that is in charge of creating and managing its _local
```
Comment (ebidel) [inline 1.0/docs/devguide/local-dom.md]: IMO, it reads better before the change. A `,` after managing might help.

The sentence would read better as a statement:
The DOM that an element creates and/or manages is called its _local DOM_.

Comment (robdodson) [inline 1.0/docs/devguide/local-dom.md]: +1 for Eric's suggestion of `The DOM that an element creates and/or manages is called its local DOM.`

**Comment by BenoitPerrot** [comment]
Ha, indeed! I read the initial sentence too quickly. Let me update patch with your suggestion.

**Comment by arthurevans** [comment]
I'd go with "and" rather than "and/or", otherwise that would be good.

**Comment by BenoitPerrot** [comment]
(done)

**Pull request merged**